packages/rscontext_3dep/.gitignore

```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
No harm in this tool having its own gitignore but it should be covered by the one in the root folder.